<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppWidget</class>
 <widget class="QWidget" name="AppWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(57, 179, 255);</string>
   </property>
   <property name="text">
    <string>常见应用</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="StyledButton" name="btnCar" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>91</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="StyledButton" name="btnRepair" native="true">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>91</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="StyledButton" name="btnBuy" native="true">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>81</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="StyledButton" name="btnPrint" native="true">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>101</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="StyledButton" name="btnAsset" native="true">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>101</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="StyledButton" name="btnRoom" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>101</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StyledButton</class>
   <extends>QWidget</extends>
   <header>widgets/StyledButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
